patroni/ha.py

```diff
@@ -257,9 +271,15 @@ def manual_failover_process_no_leader(self):
         return self._is_healthiest_node(members, check_replication_lag=False)
 
     def is_healthiest_node(self):
+        if self.is_paused() and self.cluster.failover and not self.cluster.failover.scheduled_at:
```

Comment:
Ok, on the other hand if there is already a maser running in addition to the leader/master (therefore, at least 2 masters are running), we won't make things worse by promoting to the non-master role, moreover, promoting to the non-master role is what we want, since the second master is likely unfit to support existing replicas following it (it may not archive the WALs properly). Anyway, those decisions should be documented (hopefully those comments may help).